src/shapes/Group.ts

```diff
@@ -995,7 +560,6 @@ export class Group extends createCollectionMixin(
   >(propertiesToInclude: K[] = []): Pick<T, K> & SerializedGroupProps {
     return {
       ...super.toObject([
-        'layout',
```

Comment:
the first question i have is how someone persist the layout on object restore.
The layout influence how you draw the objects too and where they are, so in theory needs to be part of the restore.

One way i can think of is suclassing of course.
Another is making the layoutManager a  hydratable option with a string, but you have to go through some gymanstic since what you want to restore is the strategy and not the manager itself that is always the same

Reply:
I agree.
What about?
```
toJSON() {
  return { strategy: this.stategy.name }
}
```

I will try it out.
Do you want me to freeze this PR or can I push changes?

Reply:
let's freeze the PR and let's address the things we may disagree on, serialization support can be another